cime_config/tests.py

```diff
@@ -283,6 +283,14 @@
             )
         },
 
+    #e3sm MMF tests for development
+    "e3sm_v3atm_integration" : {
+        "tests" : (
+            "ERP_Ln9.ne4pg2_ne4pg2.F2010-P3",
+            "ERS_Ln9.ne4pg2_ne4pg2.F20TR-P3",
```

Comment:
The above two tests need to change the grid to ne4pg2_oQU480 as the F####- P3 compsets are using mpassi for prescribed seaice and mpassi does not work on SE grids.

Also need to change Ln9 to Ld3 because of large time stepping for ROF component at this resolution (Also see [this explanation](https://github.com/E3SM-Project/E3SM/pull/4488#issuecomment-916255747) in PR #4488 concerning this change).